src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_constructs/adf_codepipeline.py

```diff
@@ -183,6 +183,28 @@ def _generate_configuration(self):
                         .get('object_key')
                     ))
                 ),
+                "KMSEncryptionKeyARN": (
```

Comment:
Did you test this? What would happen if we don't set a value, will this continue to work?

Reply:
Yeap, I tested with and without both properties just to make sure and it deploys successfully.